Sources/Request/Request/FormParam/Form.Data.swift

```diff
@@ -11,24 +11,21 @@ public extension Form {
     struct Data: FormParam {
         private let data: Foundation.Data
         private let fileName: String
-        private let mime: String
+        private let mediaType: String
```

Comment:
I think this can be stored as a `MediaType` now:
```suggestion
        private let mediaType: MediaType
```